src/styles/index.scss

```diff
@@ -1,3 +1,90 @@
-body {
+* {
+  padding: 0;
   margin: 0;
+  box-sizing: border-box;
+}
+
+$size: 100px;
+$border: 1px solid black;
+$background: #eee;
+$text-color: #000;
+$text-size: 30px;
+$gap: 1px;
+$hover-color: #ffbfcb;
+$transition-duration: 0.5s;
+$width-calendar: 7 * $size + $gap * 6;
+$days: (
+  sun: 6,
+  mon: 0,
+  tue: 1,
+  wed: 2,
+  thu: 3,
+  fri: 4,
+  sat: 5,
+);
+
+.calendar_block {
+  width: 100%;
+  height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  @each $day, $index in $days {
+    &--start-day-#{$day} {
+      .calendar__day:first-child {
+        margin-left: ($index * ($size + $gap));
+      }
+    }
+  }
+}
+
+.calendar {
+  font-family: Arial, Helvetica, sans-serif;
+  display: flex;
+  flex-wrap: wrap;
+  width: $width-calendar;
+  gap: $gap;
+}
+
+@mixin calendar-day-styles($i) {
+  width: $size;
+  height: $size;
+  background-color: $background;
+  border: $border;
```

Comment:
Instead of using the `border` shorthand property, it's recommended to specify each property individually (`border-width`, `border-style`, and `border-color`) for better clarity and maintainability. [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules)